app/presenters/app_store/v1/nsm/claim.rb

```diff
@@ -194,6 +195,10 @@ def local_record
           @local_record ||= ::Claim.find_or_create_by!(id:)
         end
 
+        def gdpr_documents_deleted?
+          respond_to?(:gdpr_documents_deleted) && gdpr_documents_deleted
```

Comment:
this logic will potentially change depending on [this conversation](https://mojdt.slack.com/archives/C05212WRK5J/p1737561667235629)

Reply:
resolved in the PR